war-wrapper/src/main/java/org/ow2/proactive/wrapper/WrapperWebConfiguration.java

```diff
@@ -0,0 +1,106 @@
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.wrapper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.log4j.Logger;
+import org.objectweb.proactive.core.util.ProActiveInet;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+
+
+/**
+ * Created by root on 30/03/17.
+ */
+public class WrapperWebConfiguration {
+
+    protected static final String REST_CONFIG_PATH = "/config/web/settings.ini";
+
+    private static final Logger logger = Logger.getLogger(WrapperWebConfiguration.class);
+
+    protected WrapperWebConfiguration() {
+
+    }
+
+    public static String getStartedUrl() {
+
+        try {
+
+            String paHost = ProActiveInet.getInstance().getHostname();
+
+            Properties properties = readWebDeploymentProperties();
+
+            int httpPort = Integer.parseInt(properties.getProperty("ear.wrapper.target.server.http.port", "9080"));
+            int httpsPort = Integer.parseInt(properties.getProperty("ear.wrapper.target.server.https.port", "9443"));
+
+            boolean httpsEnabled = isHttpsEnabled(properties);
+
+            int port = httpPort;
+
+            String httpProtocol;
+
+            if (httpsEnabled) {
+                httpProtocol = "https";
+                port = httpsPort;
+            } else {
+                httpProtocol = "http";
+            }
+
+            return httpProtocol + "://" + paHost + ":" + port;
+
+        } catch (Exception e) {
+            logger.warn("Could not find the getStarted URL", e);
+            return null;
+        }
+    }
+
+    private static boolean isHttpsEnabled(Properties properties) {
+        return "true".equalsIgnoreCase(properties.getProperty("ear.wrapper.https.enabled", "false"));
+    }
+
+    private static String getSchedulerHome() {
+        if (PASchedulerProperties.SCHEDULER_HOME.isSet()) {
+            return PASchedulerProperties.SCHEDULER_HOME.getValueAsString();
+        } else {
+            return ".";
```

Comment:
Yes I need to add "." to the path of the scheduler/rm home, in order to comply with the existing code (namely the SchedulerStarter class). Otherwise, I have to change all the exisiting sub-paths (e.g., change "/config/network/server.ini" to "config/network/server.ini"), which may bring about further issues. I opt for your solution, (adding "static final string SCHEDULER_RM_HOME_DEFAULT = ".")